docs/tasks.md

```diff
@@ -146,6 +146,67 @@ or container images that you define:
   image in the Task, rather than requesting the sum of all of the container
   image's resource requests.
 
+#### Step Script
```

Comment:
oh hai @ImJasonH could you add a link to this at the top of the file somewhere? (TOC)

Reply:
https://github.com/tektoncd/pipeline/pull/1462